augur/tasks/git/util/facade_worker/facade_worker/analyzecommit.py

```diff
@@ -356,19 +318,10 @@ def store_commit(repos_id,commit,filename,
 					whitespaceCheck.append(line[1:].strip())
 
 	# Store the last stats from the git log
-	store_commit(repo_id,commit,filename,
+	recordToInsert = generate_commit_record(repo_id,commit,filename,
 		author_name,author_email,author_date,author_timestamp,
 		committer_name,committer_email,committer_date,committer_timestamp,
 		added,removed,whitespace)
```

Comment:
This appears to only take the last file encountered in the git log since the recordToInsert variable is being overwritten for each file found. I could be misunderstanding the parsing, but recordToInsert is be overwritten by the last call (the one outside the for loop) to generate_commit_record. So all the calls to generate_commit_record inside the for loop won't matter

Reply:
Oh I don't know how I didn't notice that. I will make the method return a list next time with all of the files included.